projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -1,15 +1,23 @@
 import { Component, Element, Event, EventEmitter, Listen, Prop, State } from '@stencil/core';
 import { DotCMSContentTypeField, DotFieldStatus, DotCMSKeyValueField } from '../../models';
 import { DotFormFields } from './dot-form-fields';
-import { getClassNames, getOriginalStatus, getStringFromDotKeyArray, updateStatus } from '../../utils';
+import {
+    getClassNames,
+    getOriginalStatus,
+    getStringFromDotKeyArray,
+    updateStatus
+} from '../../utils';
 
 const fieldParamsConversionToBE = {
     'Key-Value': (values: string): { [key: string]: string } => {
         const returnValue = {};
-        values.split(',').forEach((item: string) => {
-            const [key, value] = item.split('|');
-            returnValue[key] = value;
-        });
+
+        if (typeof values === 'string' && values) {
```

Comment:
test?

Reply:
When someone work on all the tests in the dot-form